vpd-tool/src/vpd_tool_main.cpp

```diff
@@ -0,0 +1,97 @@
+#include <CLI/CLI.hpp>
+
+#include <filesystem>
+#include <iostream>
+
+int main(int argc, char** argv)
+{
+    int l_retValue = 0;
+    try
+    {
+        CLI::App l_app{
+            "VPD Command line tool to dump the inventory, read and "
+            "update the keywords,\nto fix mismatch keyword's and reset keyword's value to its default.\n"};
+
+        std::string l_objectPath{};
```

Comment:
instead of l_objectPath, can we have l_vpdPath .. why because this -O takes either objectpath/filesystem path..

and now we can't change the option name we provide to the external user. Let it be -O/--object only.